frontends/infinite-corridor/src/api/fields/factories.ts

```diff
@@ -1,7 +1,8 @@
 import { faker } from "@faker-js/faker"
```

Comment:
```suggestion
import { faker } from "@faker-js/faker/locale/en"
```

Reply:
_Aside for other mitodl folks: Shinigami92 is one of the faker-js maintainers [previously noted](https://github.com/mitodl/open-discussions/issues/3714#issuecomment-1239102062) that using specific imports like this would reduce faker's footprint._

Personally, I'm somewhat inclined to keep the faker imports as is because:
- faker should not influence the overall bundle size at all. The fact that faker was influencing our bundle size prior to this PR was a mistake.
- the subpath import is a little more verbose. I'm not really confident that we'd use it consistently, and per ☝️ I do not understand the benefit.

Shinigami92: If I've misunderstood and there is some benefit I'm not seeing, feel free to let me know.

Reply:
No you are right :+1: 
I would suggest to upgrade to v8 as soon as it's available